packages/ckeditor5-theme-lark/theme/ckeditor5-table/tablelayout.css

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+:root {
+	--ck-table-layout-widget-type-around-button-size: 16px;
+	--ck-table-layout-widget-type-around-icon-width: 10px;
+	--ck-table-layout-widget-type-around-icon-height: 8px;
+	--ck-table-layout-default-border-color: hsl(0, 0%, 58%);
+	--ck-table-layout-widget-handler-icon-size: 10px;
+}
+
+.ck-editor__editable {
+	& .table.layout-table {
+		--ck-widget-type-around-button-size: var(--ck-table-layout-widget-type-around-button-size);
+
+		& > table {
+			border-collapse: separate;
+		}
+
+		/* Removed default background color from focused table cell */
+		& > table > tbody > tr > td {
+			--ck-color-selector-focused-cell-background: transparent;
+			box-shadow: none;
+			padding: 0px 2px;
+
+			&[style^="width:"],
+			&[style*=" width:"],
+			&[style*=";width:"],
+			&[style*="'width:"] {
+				min-width: auto;
+			}
+		}
+
+		/* Default table layout stylization in editing view when border is not set.
+		But it need to be more strict cause of style attributes like `border-collapse`, `border-spacing`, etc. */
+		& > table,
+		& > table > tbody > tr > td {
+			&:not(
+			[style*="border:"],
+			[style*="border-top"],
+			[style*="border-bottom"],
+			[style*="border-left"],
+			[style*="border-right"],
+			[style*="border-width"],
+			[style*="border-style"],
+			[style*="border-color"]) {
+				border-color: transparent;
+				border-width: 0;
+				outline: var(--ck-table-layout-default-border-color) 1px dashed;
+
+				&:focus {
+					outline: var(--ck-color-focus-border) 1px solid;
+				}
+			}
+		}
+
+		/* Widget type around overrides. */
+		&.ck-widget {
+			& .ck-widget__type-around__button svg {
+				width: var(--ck-table-layout-widget-type-around-icon-width);
+				height: var(--ck-table-layout-widget-type-around-icon-height);
+			}
+
+			& > .ck-widget__type-around {
+				& > .ck-widget__type-around__button {
+					&.ck-widget__type-around__button_before,
+					&.ck-widget__type-around__button_after {
+						transform: translateY(0);
+					}
+
+					&.ck-widget__type-around__button_before {
+						border-radius: 0 0 100px 100px;
+					}
+
+					&.ck-widget__type-around__button_after {
+						border-radius: 100px 100px 0 0;
+					}
+				}
+			}
+
+			&.ck-widget_with-selection-handle {
+				--ck-widget-handler-icon-size: var(--ck-table-layout-widget-handler-icon-size);
+
+				& > .ck-widget__selection-handle {
+					transform: translateY(calc(0px - var(--ck-widget-outline-thickness)));
+					z-index: var(--ck-z-default);
+				}
+
+				& > .ck-widget__type-around {
+					& > .ck-widget__type-around__button_before {
+						margin-left: var(--ck-table-layout-widget-type-around-button-size);
+						left: min(10%, 30px);
+					}
+				}
+			}
+
+			&.ck-widget.ck-widget_selected {
```

Comment:
Probably hover state is not handled as it looks like a circle gradient when widget has yellow outline.